src/amp-user-notification(with_backend_service).html

```diff
@@ -0,0 +1,80 @@
+<!--
```

Comment:
Could we put this in the same sample and show both dialogs above each other? Each one could say which version it is.